core/spec/models/spree/taxon_spec.rb

```
@@ -783,6 +783,35 @@
   end
 
+  describe '#page_builder_image' do
+    subject(:page_builder_image) { taxon.page_builder_image }
+
+    let(:taxon) { build(:taxon, image: image, square_image: square_image) }
+
+    context 'when image and square image are not attached' do
+      let(:image) { nil }
+      let(:square_image) { nil }
+
+      it { is_expected.to_not be_attached }
```

RuboCop (1.75.5)

[convention] Prefer `not_to` over `to_not`.

(RSpec/NotToNot)